### DISCUSSION TITLE
科技產業真的有那麼好嗎?

### DISCUSSION THREAD

**Discussion by Superlan8** [1101-GS4719-C/Python-Community-GS4719-C#1165] [General]
大家好，我是資工二的莊秉郁，今天想跟大家談談最近的科技產業的裁員潮。

今年初，許多美國大型科技產業開始大幅裁員，包括Amazon, Microsoft, Google等都已經解雇了數千名員工。然而實際去探討其中的原因，發現這些裁員並非因為公司虧損所導致的，最主要的原因歸咎疫情時代的大規模招聘熱潮，為了爭奪頂尖人才，薪水也創下了新高，但隨著疫情趨緩，經濟不佳、美元急遽升值、烏俄戰爭導致的能源價格上升，這些原因使大部分科技公司發現當初在疫情時期過度擴張，誤判了科技業的發展前景，這些經濟的不確定性大大影響了科技公司的政策決定。

然而這些跟我們學生有甚麼相關呢?如果大家有在關注近幾年的研究所版或科技板等相關論壇，可以發現想往科技產業進軍的人大幅增加，許多人都想趁著這波熱潮來分一杯羹，但是當市場逐漸穩定，所需的人才遠遠小於供給時，這杯羹也就沒那麼容易分到了。

這篇文章不是因為我是資工然後來勸退其他人的，只是想分享其實科技業已經沒像大部分人想的那麼簡單了，大家可以去看一些國際新聞，就能知道其實這波裁員潮比我講的還要嚴重很多，不是單純靠個學歷就能獲得你理想中的工作，想要在當中立足還需要很多努力，這文章不只寫給大家，也寫給我自己，鼓勵大家一起加油。

**Comment by xxrjun** [comment]
這波大規模裁員真的很嚴重，然而大規模裁員也有可能不是科技公司誤判，GitHub前CEO於去年10月公開在Twitter說過 **"許多科技公司都超編了 2-10 倍，每個關注的人都知道這一點"** ，並解釋 **"當資金自由流動時，僱用更多人比處理低績效者更容易"**。 意思是當景氣好或是新技術、新業務出現時，公司肯定會招更多員工，冗員少不了，景氣沒這麼好時自然會大裁員。

我覺得聽起來既合理但又蠻悲哀的...因為這樣對於員工來說掌握度實在太低。景氣好時，科技公司(勢必得?)搶人才，造成過度擴偏，身為員工很可能被迫成為冗員(因為工作量不需要這麼多人)，然後景氣不好或是派對逐漸冷卻時，這些員工將會是第一波被裁員的。如果是新進的員工，本以為搭上順風車找到好工作，結果進公司一段時間你能貢獻的不多，然後 就被裁了==。實際情況我也不太確定是不是這樣，只能說大家加油，我也要加油。

![image](https://user-images.githubusercontent.com/40348319/227741020-fd1ef30d-60ea-4946-8466-18ad8d96952b.png)

**Comment by Jack-1D** [comment]
我認為這個裁員潮是必須的，畢竟資工市場已經處於供過於求的狀態了。即使大部分的人都看好科技的飛速進步一定會帶來更多的就業機會，然而當技術門檻越來越高時，多數人都只能用網路上可取得的資源做一些最基本的應用，只有少數人能夠在資工領域做出創新，並為科技進步做出實質貢獻。

**Comment by a3240281370** [comment]
目前看起來應該不只科技業有裁員潮喔，當前加息＋縮表的金融環境造成了信貸緊縮，而信貸緊縮的影響是全面性的，信貸緊縮可能導致企業裁員潮。當信貸緊縮時，企業面臨融資困難，無法獲得足夠的資金支持業務運營和擴展。為了減少成本，企業可能被迫實施裁員措施，以減少人力成本。這可能導致失業率上升，影響整體經濟和家庭收入。

**Comment by 1104030360** [comment]
我認為大家都只看到科技業高薪的那一面卻忽略了那些員工在背後付出了怎樣的辛苦才得到這份工作，可能在我們睡覺的時候他們都在想程式該如何寫比較好該如何debug，可能在我們開心打球的時候，他們必須坐在位子上刷leetcode，總之，我認為科技業不簡單。

**Comment by ALAN30359704** [comment]
我認爲科技業的淡旺季的情形較傳統產業來的明顯，因為我本身是土木系，老師們都跟我們說，越老越值錢，相較於科技業的職業生命較久，需求也較穩定，但各行有各行需面臨的問題，所以我覺得沒有標準的好，只有適不適合！

**Comment by sumairu0727** [comment]
前幾天也有跟導師談到關於未來出路的問題，他說即使電子產業現在是很大的一塊餅，但假如每個人都進來分，自己能力又只有平庸還是只能在電子產業當高價奴隸，到最後會變成為了錢在做不喜歡的事。
與其如此還是盡量去選擇自己有興趣的事，並且把它學到專精，即使這個產業不比電子產業巨大，自己也還是會有不錯的回報

**Comment by ashleyy-huang** [comment]
我認為要在科技業職場立足長就需要有特殊的技能，而且是不可以被其他人輕易取代的，不然很容易在這種大裁員風波中被公司淘汰，我認為一定要持續進修才可以加強自己的能力，並把握機會向老闆證明自己是個人才
。

**Comment by larry40305** [comment]
單憑學歷並不能保證你能獲得理想中的工作，要在科技領域立足仍然需要付出很多努力。

**Comment by zhiron544** [comment]
我認為作者對於資工市場的判斷是很有道理的，科技產業不是一個容易進入和立足的領域，需要不斷地學習和創新才能在競爭中生存。我也同意作者對於GitHub前CEO的觀點，科技公司的招聘政策確實對於員工來說很不公平，讓他們成為了可有可無的棋子。我希望作者能夠繼續努力，不要被裁員潮打敗，找到自己真正喜歡和擅長的方向。

**Comment by Alex11392** [comment]
但如果追求的目標是進入科技業是因為那是自己洗且適合的工作內容，而不是因為他的薪水，那依然是很值得努力的目標。

**Comment by ThanatosJun** [comment]
樓上說「追求自己喜歡且適合的工作內容是值得努力的目標」，我認為的確，但我也碰過一個問題，適合與喜歡是兩回事，我可能喜歡但我並不適合，不管在怎麼努力都追趕不上他人的腳步，這種無力感會壓垮信心，導致放棄夢想，最後圖個清靜。現在的科技產業是紅海，每一位在此產業下的社會人都得拼命的競爭，爭奪資源。好一點的開創出新的海洋，構成藍海，然而在壓力下，這片新海不出幾年，或數個月便已經慢慢染紅了。這無限循環我至今仍無解，因為競爭就是人狩獵的本能吧！
